site/src/components/gradients.md

```diff
@@ -17,12 +17,19 @@ There are two main types of gradients.
 
 ## Text color gradients
 
-Currently there is only one available gradient for text. Because gradients transitions take up the entire width of a particular element, it's recommended to highlight inline elements or a portion of a heading, not the entire heading itself.
+Gradients transitions take up the entire width of a particular element, it's recommended to highlight inline elements, icons, or a portion of a heading, not the entire heading itself.
```

Comment:
```suggestion
Because gradient transitions take up the entire width of a particular element, it's recommended to highlight inline elements, icons, or a portion of a heading, and not the entire heading itself.
```

Reply:
Thanks, combined with @homing1 punctuation feedback.